daemon/daemon.go

```diff
@@ -105,14 +105,12 @@ func checkWaitApConnect() bool {
 func checkManualMode() bool {
```

Comment:
actually all or most of daemon functions should have unit tests. I'm refactoring now to do this and will include this in the revised PR